Microsoft.DotNet.Interactive/FormattedValue.cs

```diff
@@ -40,15 +40,12 @@ public static IReadOnlyCollection<FormattedValue> FromObject(object value)
 
         private static string MimeTypeFor(Type returnValueType)
```

Comment:
This whole method should really move into `Formatter` and be configurable so that special knowledge of the types is configurable by extension authors.